RUFAS/routines/animal/ration/ration_optimizer.py

```diff
@@ -945,8 +965,15 @@ def optimize(
             )
             x0 = [np.mean(bnd) for bnd in bnds]
         else:
-            bnds = []
-            bnds = [(0, (lim / 3) + 0.0001) for lim in ration_config.feed_limit_list]
+            # bnds = []
+            # bnds = [(0, (lim / 3) + 0.0001) for lim in ration_config.feed_limit_list]
+            bnds = list(zip(
+                [(lim / 3) for lim in ration_config.feed_minimum_list],
+                [(lim / 3) for lim in ration_config.feed_limit_list]))
```

Comment:
```suggestion
            lower_bounds = [(lim / 3) for lim in ration_config.feed_minimum_list]
            upper_bounds = [(lim / 3) for lim in ration_config.feed_limit_list]
            bnds = list(zip(lower_bounds, upper_bounds))
```
Why is there a / division by 3 here? It would be good to stick this value in variable and name it to make it clearer,  if possible.

Reply:
Excellent question! https://github.com/RuminantFarmSystems/MASM/pull/1846

Older code, and the reasoning has been deprecated. I'll have to flesh out the description in 1846, which was a test to ensure that the tripling wasn't happening for valid/extant reasons.